src/components/artwork_grid.tsx

```diff
@@ -66,10 +66,8 @@ export class ArtworkGrid extends React.Component<GridProps, null> {
       for (let j = 0; j < sectionedArtworks[i].length; j++) {
         const artwork = sectionedArtworks[i][j]
         artworkComponents.push(
-          <Artwork
-            artwork={artwork}
-            key={"artwork-" + j + "-" + artwork.__id}
-          />)
+          <Artwork artwork={artwork as any} key={"artwork-" + j + "-" + artwork.__id} />,
```

Comment:
I'm curious about this from a stylistic standpoint. I find the former easier to read, is there a reason for making it one line?